# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/firefox/hotfix-armeb.patch
# Copyright (C) 2023 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- firefox-107.0/build/moz.configure/init.configure.vanilla	2022-11-17 17:55:31.419221937 +0100
+++ firefox-107.0/build/moz.configure/init.configure	2022-11-17 17:57:55.802208026 +0100
@@ -529,7 +529,7 @@
         endianness = "big"
     elif cpu.startswith("arm"):
         canonical_cpu = "arm"
-        endianness = "big" if cpu.startswith(("armeb", "armbe")) else "little"
+        endianness = "big" if cpu.endswith(("eb", "be")) else "little"
     elif cpu in ("m68k"):
         canonical_cpu = "m68k"
         endianness = "big"
